README.md

```diff
@@ -1 +1,55 @@
-# todo-List
\ No newline at end of file
+# todoList-with-webpack
+
+## Project Description
+ > "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. You will build a simple website that allows for doing that, and you will do it using ES6 and Webpack!
+
+
+## Getting Started
+> If you want a copy of this file go to the github repository and download it. 
+
+<!-- - [`git clone https://github.com/alexander16108/Chaws-Awesome-B00k-Store/`](https://github.com/alexander16108/Chaws-Awesome-B00k-Store.git) -->
+
+After cloning it you can see the project opening the index.html file with your favorite browser 
+
+
+## Screenshot of the page
+
+### Live Version Link
+
+ [Live Version]( https://alexander16108.github.io/todo-List/dist/)
+
+
+### Build With
+ - HTML
+ - JAVASCRIPT
+ - CSS
+ - WEBPACK
+
+
+
+### Contributors
+
+***👤Alexander Mayowa Odufuye***
+
+ GitHub : [@alexander16108](https://github.com/alexander16108)
+ 
+ LinkedIn : [Alexander Odufuye]()
+
+### 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+#### Show your support
+Give a ⭐️ if you like this project!
+
+#### Acknowledgments
+- Microverse.org for assigning us this project
+
+- The Amazing staff for working round the Clock to make our journey free of bumps.
+
+- The amazing code reviewers for making us improve every day :thumbsup:
+
+### Licence 
+
+This Project is MIT licensed
```

Comment:
[OPTIONAL]
- Provide a link to your License.

Reply:
ok thanks i will